adlik_serving/clients/task/build_pip_package.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
```

Comment:
Copyright & license should be added.

Reply:
copyright checker doesn't check shell script